src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1717,6 +1782,23 @@ PUBLIC_API STATUS signalingClientGetStateString(SIGNALING_CLIENT_STATE, PCHAR*);
  */
 PUBLIC_API STATUS signalingClientDeleteSync(SIGNALING_CLIENT_HANDLE);
 
+/**
+ * @brief Get the relevant/all metrics based on the RTCStatsType field. This does not include
+ * any signaling related metrics
+ *
+ * @param PRtcPeerConnection Peer connection for which the stats need to be collected
+ * @param[in/out] PRtcStats The stats object with the RTCStatsType field populated
+ */
+PUBLIC_API STATUS getRtcPeerConnectionStats(PRtcPeerConnection, PRTCStats);
+
+/**
+ * @brief Get signaling related metrics
+ *
+ * @param[in] SIGNALING_CLIENT_HANDLE Signaling client handle
+ * @param[in/out] PSignalingClientMetrics Signaling stats
+ */
+PUBLIC_API STATUS getSignalingClientStats(SIGNALING_CLIENT_HANDLE, PSignalingClientMetrics);
```

Comment:
To be consistent with naming:

* Please move this API together with the signaling APIs
* Rename to signalingClientGetMetrics